app/containers/TwoFactor/index.tsx

```diff
@@ -9,21 +9,30 @@ import { connect } from 'react-redux';
 import TextInput from '../TextInput';
 import I18n from '../../i18n';
 import EventEmitter from '../../utils/events';
-import { withTheme } from '../../theme';
+import { useTheme } from '../../theme';
 import { themes } from '../../constants/colors';
 import Button from '../Button';
 import sharedStyles from '../../views/Styles';
 import RocketChat from '../../lib/rocketchat';
 import styles from './styles';
+import { IApplicationState } from '../../definitions';
 
 export const TWO_FACTOR = 'TWO_FACTOR';
 
-interface ITwoFactor {
-	theme?: string;
-	isMasterDetail: boolean;
+interface IMethodsProp {
+	text: string;
+	keyboardType: string;
+	title?: string;
+	secureTextEntry?: boolean;
+}
+interface IMethods {
+	[key: string]: any;
```

Comment:
for the reviewer: 
I tried change this `any` to `string`, `keyof IMethods`, `IMethodsProp` and `object` but got some errors on the file. If you have other idea will be great!!